src/main/exceptionHandler.js

```diff
@@ -0,0 +1,96 @@
+// Based on electron-unhandled by sindresorhus:
+//
+// MIT License
+// Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com)
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import { app, clipboard, dialog, ipcMain } from 'electron'
+import { log } from './utils'
+import { createAndOpenGitHubIssueUrl } from './utils/createGitHubIssue'
+
+const EXIT_ON_ERROR = !!process.env.MARKTEXT_EXIT_ON_ERROR
```

Comment:
how and when need to set the `MARKTEXT_EXIT_ON_ERROR` value?

Reply:
It's used for non-interactive terminals (e.g. on AppVeyor or Travis). I change this in another PR because almost all tests fail and some CI fine tuning - on the weekend or so.

- UNIX: `export MARKTEXT_EXIT_ON_ERROR=1` in terminal or `.bashrc`
- Windows: `set MARKTEXT_EXIT_ON_ERROR=1` or environment variable GUI

or any other value which is not `false`.